spec/features/stations_index_spec.rb

```diff
@@ -14,4 +14,26 @@
       expect(page).to have_content("Union Station")
     end
   end
+
```

Comment:
I would make sure to get 2 stations in this test to confirm that you get the collection and not just a single station in the view.